backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglAWTInput.java

```diff
@@ -828,4 +828,22 @@ public void getRotationMatrix (float[] matrix) {
 		// TODO Auto-generated method stub
 
 	}
+
+	@Override
+	public float getGyroscopeX () {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+
+	@Override
+	public float getGyroscopeY () {
+		// TODO Auto-generated method stub
+		return 0;
+	}
+
+	@Override
+	public float getGyroscopeZ () {
+		// TODO Auto-generated method stub

```

Comment:
IMHO those TODOs are also not necessary. `isGyroscopeAvailable` returns false, so these won't need to be implemented as well.